server/index.js

```diff
@@ -30,6 +31,53 @@ app.get('/api/users', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.post('/api/messages', (req, res, next) => {
+  const newMessage = req.body;
+  const sender = parseInt(newMessage.senderId, 10);
+  const recipient = parseInt(newMessage.recipientId, 10);
```

Comment:
Here you can just hard-code `sender` to be `2` for now, until you have user accounts.